### PR TITLE
getedid: adding support for AMD GPUs

### PR DESCRIPTION
**Pull request by DaVukovic**
Please do not merge yet. 

This has to be tested before and for the reason I don't have some AMD GPU, I can't be sure. So please understand this PR as a request for a review.

There's also some issue with kernel 4.14 and kabylake hardware which either should be resolved or we should do changes for not supporting > kabylake. Only if newer hardware has the same problem ofc.

### CODE REVIEW

**Comment by evelikov** [comment]
General gripe/suggestion:

At some point check_gpu should be taught about "3D controller" alongside VGA - here is one example [1]. I think there are a few other things to grep for - don't recall off-hand.

https://askubuntu.com/questions/733183/difference-between-vga-compatible-controller-and-3d-controller

**Comment by DaVukovic** [comment]
I kind of agree. Those 'should' generally be used on lapops only, but I tested some things and probably found a way to detect the description of available controllers via 'lshw -c display | grep -i "description"' on my Ubuntu machine. Just need to test if that will work on LE as well. 

Thanks for the suggestion. Much appreciated.

**Comment by chewitt** [comment]
@DaVukovic any progress on this PR?

File: packages/sysutils/busybox/scripts/getedid [hunk 1]
```diff
@@ -44,8 +44,8 @@ help() {
 
 # check for GPU and store string
 check_gpu() {
-  if lspci | grep -i vga | grep -i -q intel ; then
-    gpu="intel"
+  if lspci | grep -i vga | grep -i -q 'intel\|amd' ; then
```
Comment (MilhouseVH) [inline packages/sysutils/busybox/scripts/getedid]: `grep -i -q -E 'intel|amd'` might be better?

**Pull request closed without merging**